Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: requirements
+
+# pip-compile is a part of pip-tools
+
+requirements:
```

Comment:
Is this a generated makefile?

Reply:
No. I created it to make the process of generating the requirements files and their pinned hashes from the source `requirements.in` files. If there's an update in the future to the dependencies, the update will go into the `requirements.in` file and running `make requirements` will freeze the package versions in the corresponding `requirements.txt` file for both python3 and python2 versions.

Reply:
Cool.  So I'm guessing we might wanna change the filename to be e.g. `requirements.mk` so it doesn't seem like it's used for building JM,

Reply:
Agreed.